core/components/menu/menu.js

```diff
@@ -77,27 +83,57 @@ Blockly.Menu = function() {
    * @type {?Blockly.EventData}
    * @private
    */
-  this.onKeyDownWrapper_ = null;
+  this.onKeyDownHandler_ = null;
 };
```

Comment:
Add definition for ``roleName_`` and ``element``

Reply:
Done.